examples/experimental/ProductionMonitoring/stream_table_api.ipynb

```diff
@@ -0,0 +1,170 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "25bfa9dc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "st = StreamTable(f\"{WB_ENTITY}/{WB_PROJECT}/{STREAM_TABLE_NAME}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "073d25c9",
+   "metadata": {},
+   "source": [
+    "## Step 2: Log some data\n",
+    "\n",
+    "To add rows to the StreamTable, call `.log()` on the StreamTable object. \n",
+    "`.log()` accepts a single dictionary or a list of dictionaries, where each dictionary entry corresponds to one row of the table. In each dictionary, the keys are column names and the values are the corresponding cell values.\n",
+    "\n",
+    "```python\n",
+    "st.log({\"one_column_name\" : \"value_a\", \"another_column_name\" : 7})\n",
+    "st.log([{\"one_column_name\" : \"value_b\", \"another_column_name\" : 19},\n",
+    "        {\"one_column_name\" : \"value_c\", \"another_column_name\" : 28},\n",
+    "        {\"one_column_name\" : \"value_d\", \"another_column_name\" : 36}]\n",
+    "```\n",
+    "\n",
+    "The first call to `.log()` will return a Weave Panel URL, where you can view, edit, and save the resulting StreamTable as a Weave Board, of the form:\n",
+    "\n",
+    "View data at: https://weave.wandb.ai/?exp=get%28%0A++++%22wandb-artifact%3A%2F%2F%2Fstacey%2Fmesa%2Fmy_stream_table%3Alatest%2Fobj%22%29%0A++.rows\n",
+    "\n",
+    "Subsequent log calls will silently append these rows to the StreamTable instance.\n",
+    "\n",
+    "In a notebook, the StreamTable variable on a line by itself will return a Weave Panel view of the StreamTable. The StreamTable will contain all the logged columns and their values, as well as a `timestamp` column indicating when the row was logged. By default, rows will be ordered by oldest first. You can modify a StreamTable Panel as you would a wandb.Table from the UI: sort, filter, group, etc.\n",
+    "\n",
+    "**Note: Column display order is currently non-deterministic** over repeated views of the Panel. If you would like to modify the order, open the StreamTable Panel in a new window as a Board and edit/save a Board from this seed panel. There are two options to achieve this:\n",
+    "* via the weave.wandb.ai/?exp=... URL\n",
+    "* via \"Open in new tab\" arrow button, revealed in the menu when you hover on the right side of a StreamTable panel displayed in the notebok)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b928c2e0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# log data to the StreamTable as a dictionary or list of dictionaries\n",
+    "st.log({\"col_a\" : \"1\", \"col_b\" : \"17\", \"col_c\" : \"42\"})\n",
+    "\n",
+    "# show the StreamTable\n",
+    "st"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "83cec916",
+   "metadata": {},
+   "source": [
+    "## Step 3: Log more data & explore the results!\n",
```

Comment:
can we talk a bit about how multiple runs interact here, a common case is having multiple processes log to the same stream. We should explicitly call it out and explain that it works

Reply:
happy to, and I am getting the impression that we haven't solidified this yet based on Slack convos between @shawnlewis and @tssweeney  / in my exploration it doesn't work reliably yet / unclear if I should mention the underlying artifacts / I don't understand the details myself—including a top TODO, how does one start/stop logging to a single StreamTable in one process, let alone multiple processes— so I am avoiding talking about multiple processes entirely for now in this basic doc version.

Reply:
oh can you share which thread that discuss this not working?

in my view, there are 2 cases you should explicitly call out as supported:
1. starting, stopping, starting again logging to the stream table across multiple sessions works as expected and just appends (e.g. start notebook, kill kernel, re-run notebook, if the table name is stable, it just appens)
2. parallel writes at the same time and/or writers that start/stop works - e.g. a multi-process example logging to the stream table, and maybe talk a bit about how we handle sequencing (the server uses a queue and orders the concurrent messages)

Reply:
The part/pattern that's not working is specifically what happens if we have one process logging the original StreamTable data and another process that wants to add a "user_feedback" column to the StreamTable https://weightsandbiases.slack.com/archives/C03BSTEBD7F/p1689266387878339

the 1 vs 2 distinction is very helpful! I can address these separately, and we'll get a bit closer to an example of 2 with monitoring